owncloud/ios-app
# [feature/add-demo-account] Add Account: Provide Demo Account Setup

## Description
Added a new item `Demo Account` to the `Add Account` action which provides a preconfigured setup for the ownCloud demo server.
This item is only visible, if the server list is empty.
The branded login view will be used to add the demo account, which provides also further information for ownCloud and can be configured via the `Branded.plist`
On iOS 14 the new `UIMenu` will be used, on iOS 13 `UIAlertViewController` will be used instead to present the account items.

## Related Issue
#512 

## Motivation and Context
There are also users, which are interested in ownCloud and which wants to test the ownCloud iOS app, but without having an own ownCloud instance.

## How Has This Been Tested?
- Tap on `Add account`, if the server list is empty
- Select `Demo Account`
- Follow the setup instructions
- If a bookmarks was configured, the action `Demo Account` should no longer appear

## Screenshots (if appropriate):

| ![Simulator Screen Shot - iPhone 12 Pro Max - 2020-10-28 at 09 01 06](https://user-images.githubusercontent.com/736109/97409749-6c242500-18fe-11eb-9e47-16a569d4a37b.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2020-10-28 at 09 01 11](https://user-images.githubusercontent.com/736109/97409788-79d9aa80-18fe-11eb-87e7-6ab8b1644672.png) |
|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| ![Simulator Screen Shot - iPhone 12 Pro Max - 2020-10-28 at 09 02 59](https://user-images.githubusercontent.com/736109/97409848-8fe76b00-18fe-11eb-8582-828956df5ec9.png) | ![Simulator Screen Shot - iPhone 12 Pro Max - 2020-10-28 at 09 01 20](https://user-images.githubusercontent.com/736109/97409824-86f69980-18fe-11eb-822a-b7df2b139d24.png) |

## Types of changes


## Checklist:
